.github/workflows/pythonapp.yml

```diff
@@ -18,31 +18,31 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
```

Comment:
I'm curious,  how much slower does this make things  run?
(i'm not suggesting we remove it as we need it!)

Reply:
I don't know, I haven't test it but it should be minimal because they run concurrently. It's a bit more because 3.8 takes more than the others for some reason.